drizzlepac/haputils/catalog_utils.py

```diff
@@ -201,14 +201,17 @@ def compute_background(self, box_size, win_size,
             Inverse of the footprint_mask
 
         """
+        # Negative allowance in sigma
+        negative_sigma = -1.0
+
         # Report configuration values to log
         log.info("")
         log.info("Background Computation")
         log.info("File: {}".format(self.imgname))
         log.info("Zero threshold: {}".format(zero_percent))
         log.info("Sigma-clipped Background Configuration Variables")
-        log.info("  Negative threshold: {}".format(self.negative_threshold))
-        log.info("  Negative percent: {}".format(negative_percent))
+        log.info("  Negative percent threshold: {}".format(negative_percent))
```

Comment:
I am a little confused about this log statement.  A threshold is a limit used for discrimination, whereas the 'negative_percent' seems to report the percent (number) of pixels that are negative which would not be consistent with the limit.